scripts/clockworkTwap.ts

```diff
@@ -0,0 +1,143 @@
+        side: Side.Bid,
+        priceLots: new BN(10_000), // 1 USDC for 1 META
+        maxBaseLots: new BN(1),
+        maxQuoteLotsIncludingFees: new BN(1 * 10_000),
+        clientOrderId: new BN(1),
+        orderType: OrderType.Market,
+        expiryTimestamp: new BN(0),
+        selfTradeBehavior: SelfTradeBehavior.DecrementTake,
+        limit: 255,
+    };
+
+    const userPassQuoteAccount = await token.getOrCreateAssociatedTokenAccount(
+        provider.connection,
+        payer,
+        storedPassMarket.quoteMint,
+        payer.publicKey
+    );
+
+    const userFailQuoteAccount = await token.getOrCreateAssociatedTokenAccount(
+        provider.connection,
+        payer,
+        storedFailMarket.quoteMint,
+        payer.publicKey
+    );
+    
+    // TODO Hardcode pubkeys, if open orders account already exists
+    // let passMarketOpenOrdersAccount = new PublicKey("BTVx1LQajppoeE61gisqCJdJmTexZyxk68eXQApqawHm");
+    let passMarketOpenOrdersAccount = await openbook.createOpenOrders(
+      payer,
+      passMarket,
+      "oo"
+    );
+
+    // TODO Hardcode pubkeys, if open orders account already exists
+    // let failMarketOpenOrdersAccount = new PublicKey("4hGCJuV3bcDQ4Y9pYAutyrCphVKLUVS3WuXX8QJYqkjc");
+    let failMarketOpenOrdersAccount = await openbook.createOpenOrders(
+      payer,
+      failMarket,
+      "oo"
+    );
+
+    const pass_crank = await openbookTwap.methods
+                .placeOrder(emptyBuyArgs)
+                .accounts({
+                    asks: storedPassMarket.asks,
+                    bids: storedPassMarket.bids,
+                    eventHeap: storedPassMarket.eventHeap,
+                    market: passMarket,
+                    openOrdersAccount: passMarketOpenOrdersAccount,
+                    userTokenAccount: userPassQuoteAccount.address,
+                    marketVault: storedPassMarket.marketQuoteVault,
+                    twapMarket: passMarketTwap,
+                    openbookProgram: OPENBOOK_PROGRAM_ID,
+                })
+                .instruction();
+    const fail_crank = await openbookTwap.methods
+                .placeOrder(emptyBuyArgs)
+                .accounts({
+                    asks: storedFailMarket.asks,
+                    bids: storedFailMarket.bids,
+                    eventHeap: storedFailMarket.eventHeap,
+                    market: failMarket,
+                    openOrdersAccount: failMarketOpenOrdersAccount,
+                    userTokenAccount: userFailQuoteAccount.address,
+                    marketVault: storedFailMarket.marketQuoteVault,
+                    twapMarket: failMarketTwap,
+                    openbookProgram: OPENBOOK_PROGRAM_ID,
+                })
+                .instruction();
+    
+    const threadId = failMarket.toString().substring(0, 8);
+    const [threadAddress] = clockwork.getThreadPDA(payer.publicKey, threadId);
+    const trigger = {
+        cron: {
+            schedule: "*/10 * * * * * *",
```

Comment:
would suggest doing once a minute (or once every 15 seconds if that is supported)
everyone is happy to pay the fees